lynx/src/main/java/com/github/pedrovgs/lynx/LynxShakeDetector.java

```diff
@@ -32,9 +32,11 @@
   private final Context context;
 
   private static boolean isEnabled = true;
+  private static boolean keepAlwaysActive;
 
-  public LynxShakeDetector(Context context) {
+  public LynxShakeDetector(Context context, boolean keepAlwaysActive) {
```

Comment:
As this class is public adding a new param to the constructor without keeping the previous one means that we are introducing a breaking change. To improve the library adoption for new versions we should keep the old constructor calling this one with the default expected value.